extensions/azurePublishNew/src/components/provisioningWizards/HandOffToAdminWizard.tsx

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import * as React from 'react';
+import { usePublishApi } from '@bfc/extension-client';
+import formatMessage from 'format-message';
+
+import { WizardStep, Wizard } from '../shared/wizard';
+
+import { HandOffInstructionsStep } from './steps';
+
+type Props = {
+  onStepChange: (stepId: string) => void;
+  stepIndex?: number;
+};
+export const HandOffToAdminWizard = React.memo((props: Props) => {
+  const { stepIndex, onStepChange } = props;
+  const [steps, setSteps] = React.useState<WizardStep[]>([]);
+  const { setTitle, onBack } = usePublishApi();
+  const setDialogTitle = (step: WizardStep) => {
```

Comment:
I dont understand the use of this callback, onRenderHeader is supposed to be a custom renderer for header, not a side effect.

Reply:
I have moved this to onStepChange instead.